memo.rb

```diff
@@ -1,22 +1,21 @@
 # frozen_string_literal: true
 
-require 'json'
-
 class MemoDB
-  JSON_FILE = 'data.json'
-  HIGHEST_ID = 'highest_id.json'
+  MEMO_DB = PG.connect(dbname: 'memo_db')
```

Comment:
MemoDBクラスのMEMO_DBは名前的に近すぎてちょっと紛らわしいですね。
https://www.rubydoc.info/gems/pg/PG/Connection のサンプルコードにもありますが、慣習的にCONNのような名前がよく使われます

```suggestion
  CONN = PG.connect(dbname: 'memo_db')
```

Reply:
これは自分でも紛らわしいと思っていました。　慣習的な名前も意識的に使っていくようにしたいです。